app/typography/page.tsx

```
@@ -0,0 +1,109 @@
+          <Heading size="lg">Text</Heading>
+          <div className="flex flex-col gap-3">
+            <Text>Body text base (body line height).</Text>
+            <Text variant="strong">Body text strong.</Text>
+            <Text variant="emphasis">Body text emphasis.</Text>
+            <Text variant="link" as="a" href="#">
+              Body text link
+            </Text>
+            <Text variant="input" lineHeight="single">
+              Input text (single line height)
+            </Text>
+            <Text variant="code">const value = "code";</Text>
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)